packages/spatial/src/renderer/components/MeshComponent.ts

```diff
@@ -37,7 +38,13 @@ export const MeshComponent = defineComponent({
 
   onSet(entity, component, json) {
     setComponent(entity, ObjectComponent, json as Mesh)
+    setComponent(entity, BoundingBoxComponent, {})
```

Comment:
we changed what we wanted, please revert the changes to MeshComponent and BoundingBoxComponent

See https://github.com/ir-engine/ir-engine/pull/2018